modules/EnsEMBL/Web/Component/Gene/ComparaOrthologs.pm

```diff
@@ -64,8 +64,8 @@ sub content {
   my $availability = $object->availability;
   my $is_ncrna     = ($object->Obj->biotype =~ /RNA/);
   my $species_name = $species_defs->GROUP_DISPLAY_NAME;
-  my $strain_url   = $species_defs->IS_STRAIN_OF ? "Strain_" : "";
-  my $strain_param = $self->is_strain ? ";strain=1" : ""; # initialize variable even if is_strain is falsy to avoid warnings
+  my $strain_url   = $self->is_strain ? "Strain_" : "";
```

Comment:
On Component.pm ([here](https://github.com/Ensembl/ensembl-webcode/pull/852/files#diff-3169505c843fbc8b3e47d63e013dd0d128cce011375187f2ebb84d7ddc15d595R517)) you mentioned we need to use hub->is_strain. Should we use the same here?

Reply:
Well caught! I'll fix that.